Index: SPECS/networkmanager.spec
===================================================================
--- SPECS/networkmanager.spec	(revision 499084)
+++ SPECS/networkmanager.spec	(working copy)
@@ -19,7 +19,7 @@
 %if %{snapshot}
 Release:	%mkrel 0.%{snapshot}.1
 %else
-Release:        %mkrel 1
+Release:        %mkrel 3
 %endif
 Group:		System/Base
 License:	GPLv2+
@@ -32,6 +32,7 @@
 # Fedora patches
 Patch2:		explain-dns1-dns2.patch
 # (fhimpe) Make it use correct location for dhclient lease files
+Patch3:		NetworkManager-mdv.patch
 BuildRequires:	libnl-devel wpa_supplicant libiw-devel dbus-glib-devel
 BuildRequires:	hal-devel >= 0.5.0 nss-devel intltool
 BuildRequires:	gtk-doc ext2fs-devel
@@ -129,6 +130,8 @@
 %prep
 %setup -q -n %{rname}-%{version}
 %patch2 -p1 -b .explain-dns1-dns2
+# add mandriva configuration
+%patch3 -p1
 
 %build
 autoreconf -fis
@@ -150,7 +153,7 @@
 
 cat > %{buildroot}%{_sysconfdir}/NetworkManager/nm-system-settings.conf << EOF
 [main]
-plugins=ifcfg-fedora
+plugins=ifcfg-mdv
 EOF
 
 # create a VPN directory
